City-of-Helsinki/drupal-helfi-platform-config
# Uhf 6041 paragraphs to global

# TASKNAME [UHF-6041](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6041)

Two paragraphs that were only in etusivu are now in all of the instances landing pages content area

## What was done
* Moved target group links configs from etusivu to platform config
* Moved popular services configs from helfi_frontpage to helfi_content

## How to install
* Make sure your instance is up and running on latest dev branch.
* Test this in etusivu and in some other instance as well
* When testing in etusivu checkout this PR https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/105
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-6041-paragraphs-to-global`
 * Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-6041-paragraphs-to-global`
* Run `make drush-updb drush-cr`

## How to test
* Create a landing page and add both popular services and target group links paragraph to the content. The paragraphs should be styled and look the same as before.

## Other PRs
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/384
* https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/105
